apps/desktop/src/lib/config/uiFeatureFlags.ts

```diff
@@ -22,3 +22,8 @@ export function featureTopics(): Persisted<boolean> {
 	const key = 'feature--topics';
 	return persisted(false, key);
 }
+
+export function highlightBranchLaneContentsFeature(): Persisted<boolean> {
```

Comment:
Can we make this new addition an exported constant instead of a function? Also, I'm wondering if we can improve the name. It feels more like auto selecting input field contents rather than highlighting branch lane contents.

Reply:
yup, no problem. What do you think of `autoSelectBranchLaneContentsFeature`? It's a bit long, but gets the whole idea across imo

Reply:
Yeah that totally works for me, and I'll still revert the name change of the stacking flag so it follows that convention. However, the pedantic part of me is still objecting to the "branch lane contents" part. Imho it's neither the right subject (branch contents) nor is highlighting what we're doing. It's about _text selection_ on input field values isn't it?

Reply:
Ah right - sorry, I misread something and thought it would extend to more than one field.